terraform/aws/.gitignore

```diff
@@ -0,0 +1 @@
+*.yaml
```

Comment:
I *think* this file is no longer needed now that we're not using a template here for yaml encoding?